File: cddl/contrib/dtracetoolkit/Code/Php/func_abc.php
<?php
function func_c()
{
	echo "Function C\n";
	sleep(1);
}

function func_b()
{
	echo "Function B\n";
	sleep(1);
	func_c();
}

function func_a()
{
	echo "Function A\n";
	sleep(1);
	func_b();
}

func_a();
?>

File: cddl/contrib/dtracetoolkit/Code/Python/func_abc.py
#!/usr/bin/python

import time

def func_c():
	print "Function C"	
	time.sleep(1)

def func_b():
	print "Function B"
	time.sleep(1)
	func_c()

def func_a():
	print "Function A"
	time.sleep(1)
	func_b()

func_a()
